### DISCUSSION TITLE
401 for GET Watched

### DISCUSSION THREAD

**Discussion by Christian1313** [trakt/trakt-api#375] [Q&A] [answered]
I write an app to track the watch progress and use the api.

I managed to get an access token via device authorization.
Then I wanted to call the ```sync/watched/shows``` endpoint and got an 401 error.

First I renewed my access tokens several times, but always 401 for the sync endpoint.
Second I checked with "apiary.io" and postman always the same 401.
Then i stumbled across this discussion (https://github.com/trakt/api-help/discussions/318),
where the user just created a new app with new client and secret id.
After that his requests worked.

I did the same and only replaced the ids in my code. And it worked.
Any advice?

**Comment by Christian1313** [comment, marked as answer]
Funny thing now I reverted back to the old ids the request is working. 
So there is something strange at server side here.

**Reply by rectifyer** [reply]
I don't really know, a 401 means invalid user based auth and I haven't seen any other reason the api would return that error code.

**Reply by Christian1313** [reply]
I will watch for this error, if it re occurs. Until then the discussion is for others that run into the sam problem
